src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -1,5 +1,6 @@
 { pick, extend } = require 'underscore'
 Backbone = require 'backbone'
+qs = require 'querystring'
```

Comment:
I would use `require 'qs'` instead of this lib, we've had issues with `querystring` and certain types of parsing...unfortunately I don't have great docs/reference on the subject. If it works (since this is pretty simple parsing), then it works!

Reply:
This old-ish PR makes mention of it too: https://github.com/artsy/force-private/pull/5076

Reply:
On it 👍